### DISCUSSION TITLE
blog/2023-03-03-causal-inference-with-count-regression/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#10] [Comments]
# blog/2023-03-03-causal-inference-with-count-regression/

Part 5 of the GLM and causal inference series.

https://timely-flan-2986f4.netlify.app/blog/2023-03-03-causal-inference-with-count-regression/

**Comment by sjwild** [comment]
By now I think the tag line for your posts should be "it's all g-computation" and "remember, sometimes it's the CATE, sometimes the ATE. Please don't confuse them".
* You talk about the exponentiated mean of log variables (Age and Base). Based on the code you used, you're getting back the median on the untransformed scale. This might not be clear to readers of your post, so it is probably worth explaining.
* to account for the overdispersion in the poisson models, you could also use `glm(..., family = quasipoisson())`. The SEs won't be as wide as using robust standard errors, so I'm not sure it's worth covering in this post.
* I have seen some articles about using robust SE/quasipoisson alternative to poisson regression in a bayesian context. I will need to dig them up. I have no idea if they can be easily implemented in brms or not. Sad.

**Reply by ASKurz** [reply]
Thanks for the feedback, @sjwild. I've added a couple words about exponentiation and medians.

I concede your point about the quasipoisson, but I specifically left if out because this series is decidedly likelihoodist, which seems at odds with quasi-likelihood approaches. I even considered leaving out the sandwich standard error material on that account, but figured it wouldn't kill me to give the robust crowd a bone.

As to the robust SE's for Bayesians, is [this](file:///Users/solomonkurz/Downloads/eScholarship%20UC%20item%2069c4x8m9%20(1).pdf) the paper you were thinking about?

**Reply by ASKurz** [reply]
Shoot, it looks like the link didn't work. Here's the link to the paper I tried to refer to: https://escholarship.org/content/qt69c4x8m9/qt69c4x8m9.pdf

**Reply by sjwild** [reply]
I was thinking of this one: https://projecteuclid.org/journals/annals-of-applied-statistics/volume-4/issue-4/Model-robust-regression-and-a-Bayesian-sandwich-estimator/10.1214/10-AOAS362.full

**Reply by ASKurz** [reply]
Ha! Got it. I literally found that paper and added it to the post in the time between adding my comment and when you just responded.
